charts/base/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v2
+name: base
+description: Helm chart for deploying ingress-apisix cluster resources and CRDs
+icon: https://apache.org/logos/res/apisix/apisix.png
+keywords:
+  - ingress
+  - apisix
+  - nginx
```

Comment:
If we separate crd and ingress, had better contains `crd` in key words .

Reply:
OK.

Reply:
Added.